agent/utils/utils_test.go

```diff
@@ -21,86 +21,72 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/utils/ttime"
 	"github.com/aws/amazon-ecs-agent/agent/utils/ttime/mocks"
 	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestDefaultIfBlank(t *testing.T) {
 	const defaultValue = "a boring default"
 	const specifiedValue = "new value"
 	result := DefaultIfBlank(specifiedValue, defaultValue)
-
-	if result != specifiedValue {
-		t.Error("Expected " + specifiedValue + ", got " + result)
-	}
+	assert.Equal(t, result, specifiedValue, "Expected "+specifiedValue+", got "+result)
```

Comment:
You don't need the `"Expected "+specifiedValue+", got "+result`, since `assert.Equal` will do that for you.

Also, it should be `assert.Equal(t, expected, actual, "message")`, so you'll want to swap the order.